.github/workflows/apt.yml

```
@@ -46,11 +46,31 @@ jobs:
       - name: Build deb with Docker
         run: |
           rake apt:build APT_TARGETS=${{ matrix.rake-job }}
+      - name: Build v6 deb with Docker
+        run: |
+          git config user.email "fluentd@googlegroups.com"
+          git config user.name "Fluentd developers"
+          git am fluent-package/bump-version-v6.patch
+          rake apt:build APT_TARGETS=${{ matrix.rake-job }}
+      - name: Relocate packages
+        run: |
+          cp -r fluent-package/apt/repositories fluent-package/apt/v6repositories
+          find fluent-package/apt/repositories -name 'fluent-package_6*.deb' | xargs rm -f
+          find fluent-package/apt/repositories -name 'fluent-package-dbgsym_6*.deb' | xargs rm -f
+          find fluent-package/apt/repositories -name 'td-agent_6*.deb' | xargs rm -f
+          find fluent-package/apt/v6repositories -name 'fluent-package_5*.deb' | xargs rm -f
+          find fluent-package/apt/v6repositories -name 'fluent-package-dbgsym_5*.deb' | xargs rm -f
+          find fluent-package/apt/v6repositories -name 'td-agent_5*.deb' | xargs rm -f
```

How about using another checkout repository?

```suggestion
      - uses: actions/checkout@master
        with:
          path: v6-test
      - name: Build v6 deb with Docker
        run: |
          cd v6-test
          git config user.email "fluentd@googlegroups.com"
          git config user.name "Fluentd developers"
          git am fluent-package/bump-version-v6.patch
          rake apt:build APT_TARGETS=${{ matrix.rake-job }}
```